src/editor/overlay_widget.cpp

```diff
@@ -1311,11 +1412,10 @@ EditorOverlayWidget::draw_tile_grid(DrawingContext& context, int tile_size,
 void
 EditorOverlayWidget::draw_tilemap_border(DrawingContext& context)
 {
-  if ( !m_editor.get_selected_tilemap() ) return;
+  if (!m_editor.get_selected_tilemap()) return;
 
   auto current_tm = m_editor.get_selected_tilemap();
-  if (!current_tm)
-    return;
```

Comment:
I believe this is fine.

Reply:
It should be fine either way. The guidelines say that there are no hard rules for {} within functions, so I just went with short if() return on one line, consistently in the whole file. It was 50/50 before.